tests/LSP6KeyManager/tests/AllowedERC725YKeys.test.ts

```diff
@@ -1769,12 +1884,40 @@ export const shouldBehaveLikeAllowedERC725YKeys = (
           context.keyManager
             .connect(controllerCanSetSomeKeys)
             .execute(setDataPayload)
-        )
-          .to.be.revertedWithCustomError(
-            context.keyManager,
-            "NotAllowedERC725YKey"
-          )
-          .withArgs(controllerCanSetSomeKeys.address, testCase.datakeyToSet);
+        ).to.be.revertedWithCustomError(
+          context.keyManager,
+          "ZeroDataKeyNotAllowed"
+        );
+      });
+
+      it("should pass when trying to set an array of data keys including bytes32(0)", async () => {
```

Comment:
I think there is an error here. It states "should pass" but inside the test, it tests that it reverts.

```suggestion
      it("should revert when trying to set an array of data keys including bytes32(0)", async () => {
```